build-configs/aschaffenburg/theme/colors.ts

```diff
@@ -1,7 +1,7 @@
 import { legacyCommonContrastColors, LegacyColorsType, legacyCommonLightColors } from '../../common/theme/colors'
 
 const themeColor = '#009684'
-const themeColorLight = 'rgba(0, 150, 132, 0.5)'
+const themeColorLight = 'rgb(0, 150, 132, 0.5)'
```

Comment:
❓ Why did you change these?

Reply:
Upgrading stylelint added this new rule: https://stylelint.io/user-guide/rules/color-function-notation/ Basically `rgba` is only an alias for `rgb`.